pkg/platform/model/buildplanner/build.go

```diff
@@ -118,6 +119,11 @@ func processBuildPlannerError(bpErr error, fallbackMessage string) error {
 		if ok && code == clientDeprecationErrorKey {
 			return &response.BuildPlannerError{Err: locale.NewExternalError("err_buildplanner_deprecated", "Encountered deprecation error: {{.V0}}", graphqlErr.Message)}
 		}
+		if status, ok := graphqlErr.Extensions[statusExtensionKey].(float64); ok {
+			if status >= 400 && status < 500 {
+				return locale.NewInputError("err_buildplanner_api_input_error", "{{.V0}}: {{.V1}}", fallbackMessage, bpErr.Error())
```

Comment:
I don't think `err_buildplanner_api_input_error` exists. Least not finding it in my checkout.

Reply:
Right, it does not exist, so I'm using the `"{{.V0}}: {{.V1}}"` fallback string you see here.